drivers/hmis/spec/system/hmis/intake_assessment_spec.rb

```diff
@@ -74,7 +74,7 @@ def complete_individual_assessment
         expect(e2.intake_assessment).to be_nil
 
         fill_in 'Search Clients', with: c1.last_name
-        click_link c1.brief_name
+        find('a', text: 'View Enrollment', match: :first).trigger(:click)
```

Comment:
We lose some fidelity here -- previously we knew we were clicking on the row for `c1` (or at least, a client with their name), and now we're just clicking on the first row. Can we use the aria-label here, so we know who we're clicking on?

Reply:
It looks like we have `enable_aria_label` enabled so you may be able to just look for the `text` of the aria label directly to find the button